### PR TITLE
Feature/flash messages

### PR DESCRIPTION
**Pull request by natyork**
Create a view to render all flash messages per https://www.pivotaltracker.com/story/show/142562277

Flash messages will be displayed in a div with `class="alert alert-#{key}"` where key is the flash type, so flash messages can be styled by type.

There are no custom flash types currently defined, but if in future custom flash methods are defined, such as for https://www.pivotaltracker.com/story/show/142092195 they can be defined in the applications controller per: http://api.rubyonrails.org/classes/ActionController/Flash/ClassMethods.html

### CODE REVIEW

File: app/views/layouts/_flash-messages.html.erb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<div class='alert-container'>
```
Comment (graygilmore) [inline app/views/layouts/_flash-messages.html.erb]: Should we have a conditional here like `if flash.any?`?

**Review by graygilmore — CHANGES_REQUESTED** [review]
We can squash the two commits into one as they don't make sense without each other.

**Review by jacquesporveau — APPROVED** [review]
LGTM besides Grays comment.

**Comment by natyork** [comment]
Commits squashed and conditional rendering of flash messages added per @graygilmore comments.

**Review by qr8r — APPROVED** [review]

**Comment by natyork** [comment]
@graygilmore has moved this story to the icebox. I am going to close this PR.

**Review by graygilmore — APPROVED** [review]

**Comment by graygilmore** [comment]
Merging ❤️

**Pull request merged**